### PR TITLE
add new feature for image stimuli

### PR DESCRIPTION
**Pull request by SiQube**
add a new feature which enables us to use image stimuli

### CODE REVIEW

**Comment by SiQube** [comment]
pre-commit-ci failing due to #774

File: tests/files/reading-genome-1.png
Comment (prassepaul) [inline tests/files/reading-genome-1.png:1]: do we have the copyright?

Comment (SiQube) [reply tests/files/reading-genome-1.png:1]: probably not, I'll check, I can also just use another image

**Comment by SiQube** [comment]
I changed it to a copyright free image, according to [this](https://www.pexels.com/de-de/lizenz/)  and this is the [image](https://www.pexels.com/photo/close-up-photography-of-brown-and-yellow-toy-figure-1000498/)

**Comment by SiQube** [comment]
you can checkout this branch and test the feature via:

```python
import pymovements as pm

pm.utils.plotting.draw_image_stimulus('./tests/files/pexels-zoorg-1000498.jpg', show=True)
```

**Review by prassepaul — APPROVED** [review]

**Pull request merged**